internal/db/storage/blob/s3/s3.go

```diff
@@ -17,59 +17,63 @@
 package s3
 
 import (
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/credentials"
+	"context"
+	"io"
+
 	"github.com/aws/aws-sdk-go/aws/session"
-	"gocloud.dev/blob/s3blob"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/vdaas/vald/internal/db/storage/blob"
+	"github.com/vdaas/vald/internal/db/storage/blob/s3/reader"
+	"github.com/vdaas/vald/internal/db/storage/blob/s3/writer"
+	"github.com/vdaas/vald/internal/errgroup"
 )
 
-type URLOpener = s3blob.URLOpener
-
-type sess struct {
-	endpoint        string
-	region          string
-	accessKey       string
-	secretAccessKey string
-	token           string
-	useLegacyList   bool
-}
+type s3client struct {
```

Comment:
I think s3.client is better than s3.s3client
```suggestion
type client struct {
```

Reply:
Thanks. I'll rename it.